Source/WebKit/GPUProcess/graphics/WebGPU/RemoteCompositorIntegration.messages.in

```diff
@@ -25,8 +25,9 @@
 
 messages -> RemoteCompositorIntegration NotRefCounted Stream {
 #if PLATFORM(COCOA)
-void GetRenderBuffers() -> (Vector<MachSendRight> renderBuffers) Synchronous NotStreamEncodableReply
+void RecreateRenderBuffers(int width, int height) -> (Vector<MachSendRight> renderBuffers) Synchronous NotStreamEncodableReply
 #endif
+void PrepareForDisplay() -> (bool dummy) Synchronous // FIXME: https://bugs.webkit.org/show_bug.cgi?id=250993 Wait for the results to be fully drawn
```

Comment:
should you add `NotStreamEncodableReply` since that will be needed when we wait for the results to be drawn?

Reply:
Will it be needed? None of the data structures passed in the message (either sent or replied) are unable to be represented in raw memory. I thought this was why the only message that has NonStreamEncodable is the message which replies with a MachPort (because a Mach port cannot be represented in raw memory)

Reply:
`NotStreamEncodableReply` is a workaround for asynchronous replies for synchronous messages, I will discuss offline